### PR TITLE
adding compat data for api.attr

### PR DESCRIPTION
**Pull request by katzrkool**

### CODE REVIEW

File: api/Attr.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "Attr": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Attr",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true,
+            "notes": "As of Chrome 45, this property no longer inherits from Node."
```
Comment (jpmedley) [inline api/Attr.json]: This is true of `webview_android` and `chrome_android` as well.

File: api/Attr.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            "version_added": true
+          },
+          "ie": {
+            "version_added": true
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
```
Comment (jpmedley) [inline api/Attr.json]: The Chrome note applies to Opera 32, both flavors.

**Comment by katzrkool** [comment]
That should fix it.

**Comment by Elchi3** [comment]
This misses a few sub features:
https://developer.mozilla.org/en-US/docs/Web/API/Attr/localName
https://developer.mozilla.org/en-US/docs/Web/API/Attr/namespaceURI
https://developer.mozilla.org/en-US/docs/Web/API/Attr/prefix

File: api/Attr.json [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+            "notes": "As of Opera 32, this property no longer inherits from Node."
+          },
+          "safari": {
+            "version_added": true
+          },
+          "safari_ios": {
+            "version_added": true
+          }
+        }
+      },
```
Comment (connorshea) [inline api/Attr.json]: This compat object and the subfeatures should have status info like this:

```json
"status": {
  "experimental": false,
  "standard_track": true,
  "deprecated": false
}
```

For example:

https://github.com/mdn/browser-compat-data/blob/ac286bac1a3e418d03f3e324907fdaaeb4e22dc0/api/AbortController.json#L47-L51

**Comment by katzrkool** [comment]
made the status changes

**Comment by connorshea** [comment]
This is missing some subproperties but they don't have MDN articles so I don't know if we really care 🤷‍♂️

**Comment by Elchi3** [comment]
> This is missing some subproperties but they don't have MDN articles so I don't know if we really care

I think it is good enough for an initial migration for now. Thanks everyone!

**Pull request merged**